app/controllers/catalog_controller.rb

```diff
@@ -109,6 +109,7 @@ class CatalogController < ApplicationController
     config.add_show_field solr_name('desc_metadata__addinfolink', :stored_searchable, type: :string), :label => 'Additional Information',
```

Comment:
Surprised to see SDR stakeholders wanting the link to LibGuides removed, since they were the original requesters of that functionality. Given that they no longer want this, should we consider removing the addinfolink and addinfotext functionality entirely? It was only used here, and if we need it again we probably want to do it differently anyway. It has a bad smell to me -- I wrote it, so that plays in -- and I think it adds needlessly to technical debt at this point. I suggest we take it out of Nyucore and the catalog_controller completely.